frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.jsx

```diff
@@ -53,7 +59,7 @@ const optionsMessagePropTypes = {
 
 const mapDispatchToProps = {
```

Comment:
I think we can get rid of the `mapDispatchToProps` with `useDispatch` + directly importing the actions. Same for `mapStateToProps`, we can just use `useSelector` and then we can drop the `connect` HoC.

Reply:
absolutely, that's on my toDo list, but I wanted to keep the PRs as small as possible and do the minimum needed to make this a functional component.